docs/development.rst

```diff
@@ -56,45 +56,14 @@ Serve the documentation locally with this command:
 
 Now, you can hit http://localhost:9000/ and you should see your documentation here.
 
-
-Avoid CORS on local backend
-+++++++++++++++++++++++++++
-
-When building the documentation locally,
-the API calls will be blocked by the browser because of :abbr:`CORS (Cross Origin Resource Sharing)`
-(you are hitting the URL http://localhost:9000/ and the API calls are to https://readthedocs.org/).
```

Comment:
We need to support building the docs locally (via `make html`) and hitting RTD servers for Embed API.

Is this possible now? Did we enable CORS on `/api/v2/embed`? Have you tested it?

Reply:
If this works by default, we should default to `readthedocs.org` and change it to `/_` if buildinig on RTD and the setting is not defined by the user as @ericholscher said in https://github.com/readthedocs/sphinx-hoverxref/pull/134#discussion_r661846609

Reply:
CORS isn't a problem anymore. I'm not sure having dynamic settings is a good idea, users already need to change version and project slug if they want to have it working locally, and also they would need to change it to rtd.com when using commercial.